src/composer/src/build.rs

```diff
@@ -248,12 +250,58 @@ fn constructor_serialize_args(
     Ok(args_file_path)
 }
 
+fn comp_gen_make_cmd_scan_lib_constant(constants: &[String]) -> Result<Vec<String>, String> {
+    println!("Scanning libraries for constants: {:?}", constants); // Print the constants being scanned for
```

Comment:
The composer really focuses on not printing much out, so I don't like this.

We might want a verbose mode, but we don't have that yet. Don't think that the comment adds anything over the print.

Reply:
sounds good, now it is only for my debug purpose, will remove them later